sources/niCURL/src/_build.ham

```diff
@@ -12,6 +12,12 @@ if ! ( $(USE_SYSTEM_LIBCURL) = 1 ) {
   SubDirHdrs $(SUBDIR) libCURL include ;
 }
 
```

Comment:
I really dont know how to do this...I tried a lot of different ways. I just want to se extra linking flags ONLY FOR niCURL.
I tried LINKFLAGS on niCURL, LINKFLAGS on $(PKGTARGET), TargetLinkFlags $(<) : and also creating _rules.ham in the niCURL folder and overrite the flags there.

As I have it this PR it works if I run ham all inside the niCURL directory, the problem I have is that when I run ham all in the root of niLang, other projects like niUI are trying to link _FetchOverride_Error also.